packages/s/sord/package.yml

```diff
@@ -1,17 +1,19 @@
 name       : sord
-version    : 0.16.4
-release    : 4
+version    : 0.16.16
+release    : 5
 source     :
-    - git|https://github.com/drobilla/sord.git : v0.16.4
-license    : ISC
+    - https://download.drobilla.net/sord-0.16.16.tar.xz : 257f876d756143da02ee84c9260af93559d6249dd87f317e70ab5fffcc975fd0
+homepage   : https://drobilla.net/software/sord
+license    : 0BSD
```

Comment:
It looks like this project uses `0BSD`, `BSD-3-Clause`, and `ISC` [licenses](https://github.com/drobilla/sord/tree/master/LICENSES). I would guess all of them should be listed:

```yaml
license     :
    - 0BSD
    - BSD-3-Clause
    - ISC
```

Reply:
Might want to check the other related projects, too.

Reply:
Licenses fixed, the other two did not use the 3-Clause.